hepcrawl/spiders/common/oaipmh_spider.py

```diff
@@ -52,7 +52,7 @@ def __init__(
         url,
         format='oai_dc',
         sets=None,
-        alias=None,
+        harvest_single=None,
```

Comment:
what other parameters are allowed when `harvest_single` is set? does it make sense to have both `harvest_single` and `sets`, or `from_date`, or `until_date`?

Reply:
No, only `format` is allowed with `harvest_single`

Reply:
Then it would be good to check arguments and raise an exception (`ValueError`) if used incorrectly.